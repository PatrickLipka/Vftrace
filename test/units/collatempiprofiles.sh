#!/bin/bash
set -x
test_name=collatempiprofiles
configfile=${test_name}.json
output_file=${test_name}.out
ref_file=${srcdir}/ref_output/${test_name}.out

rm -f ${output_file}

echo "{\"mpi\": {\"show_table\": true, \"log_messages\": true}}" > ${configfile}
export VFTR_CONFIG=${configfile}
${MPI_EXEC} ${MPI_OPTS} ${NP} 1 ./${test_name} > ${output_file} || exit 1

diff ${output_file} ${ref_file}
