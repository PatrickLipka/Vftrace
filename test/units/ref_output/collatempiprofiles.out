func2<func1<init:
   nmsg: 2/2, msgsize: 2560/5120
func0<init:
   nmsg: 3/3, msgsize: 42/84
init:
   nmsg: 0/0, msgsize: 0/0
func1<init:
   nmsg: 1/1, msgsize: 128/256
